src/hooks/useUser.ts

```diff
@@ -45,6 +45,7 @@ export default function useUser(): { user: User } {
     getUser,
     {
       refreshInterval: 100000,
+      revalidateOnMount: true,
```

Comment:
this fixes the bug of not re-validating after we log-in even though we called `mutate`